src/components/Day/TodayPlan/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+import ForwardEmojiPicker from './EmojiPicker';
+import TodayPlanInput from './TodayPlanInput';
+
+function TodayPlan() {
+  const [click, setClick] = useState<boolean>(false);
+  const useOutsideAlert = (ref: React.RefObject<HTMLDivElement>) => {
+    useEffect(() => {
+      function handleClickOutside(event) {
+        if (ref.current && !ref.current.contains(event.target)) {
+          setClick(false);
+        }
+      }
+      document.addEventListener('mousedown', handleClickOutside);
+      return () => {
+        document.removeEventListener('mousedown', handleClickOutside);
+      };
+    }, [ref]);
+  };
+  const refPicker = useRef<HTMLDivElement>(null);
+  useOutsideAlert(refPicker);
+
+  return (
+    <StyledTodayPlan.Root>
+      <ForwardEmojiPicker ref={refPicker} click={click} setClick={setClick} />
```

Comment:
```suggestion
       <ForwardEmojiPicker ref={refPicker} click={click} setClick={handleClick} />
```
```
const handleClick = (value:boolean) => {
     setClick(value)
}
```
props로 setState를 그대로 넘겨주는거 보다 아래처럼 handler함수를 만들어서 setState를 wrapper하여 넘겨주는게 더 좋다고 합니다! 자식 컴포넌트에서 setState를 사용하는 부분은 handleClick(false) 와 같은 방식으로 사용하겠네요!